lib/chef/resource/windows_package.rb

```diff
@@ -26,13 +26,12 @@ class Resource
     class WindowsPackage < Chef::Resource::Package
       include Chef::Mixin::Uris
 
-      provides :package, os: "windows"
       provides :windows_package, os: "windows"
+      provides :package, os: "windows"
```

Comment:
I think that was an artifact of the changes that were mixed in earlier, so: moot.